README.md

```diff
@@ -1,4 +1,4 @@
-# `gr2m/github-app-token-action`
+# `actions/github-app-token`
```

Comment:
```suggestion
# GitHub App Token
```

This would make the header more consistent with other repos in this organization.